templates/launch.html

```diff
@@ -392,6 +392,7 @@ <h2 class="ons-details__title">Runner Metadata</h2>
         </div>
     </form>
     <script src="https://cdn.ons.gov.uk/sdc/design-system/70.0.7/scripts/main.js"></script>
+    <script>const sdsEnabled = {{ .SdsEnabled }} === "true"</script>
```

Comment:
This is getting highlighted by the linters, can we pass the Go var in quotes like in other places in this file and cast it as a bool instead of comparison?

Reply:
Which linters? I cant see anything on the lint action or anything in my IDE being highlighted